portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityAttachment.java

```diff
@@ -70,4 +74,8 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(getType(), getUrl(), getName(), getContent());
   }
+
+  @Nullable
```

Comment:
I suppose `getIdempotentId` should be `@Nonnull` here?

Reply:
Yes, updated it